src/geo/gmaps/gmaps-torque-layer-view.js

```diff
@@ -42,7 +42,7 @@ _.extend(
 
       if ('tileURLTemplates' in changed) {
         // REAL HACK
-        this.provider.templateUrl = this.model.getTileURLTemplates()[0];
+        this.provider.templateUrl = this.model.getTileURLTemplate();
```

Comment:
As I mentioned in https://github.com/CartoDB/cartodb.js/pull/1602/files#r107705091, I'm not sure Google Maps will know how to handle `{s}` placeholders...